src/FixedDataTable.js

```diff
@@ -663,9 +663,7 @@ class FixedDataTable extends React.Component {
   }
 
   componentDidUpdate(/*object*/ prevProps) {
-    if (prevProps !== this.props) {
```

Comment:
no need to check for change anymore?

Reply:
Nope, because the `this._didScroll` already compares the scroll props before dispatching actions.

Furthermore, I believe comparing just the prop object's reference in order to skip actions might lead to bugs in the future (like if the user just changed the classname prop but the action still fires because prop reference will get changed). So it's better if the action itself just selects and compares the props it needs.